### PR TITLE
✨ [PRTL-1796] added main actions to query page

### PR DESCRIPTION
**Pull request by aishmitk**

### CODE REVIEW

File: src/packages/@ncigdc/containers/SmartSearchPage.js [hunk 1]
```diff
@@ -121,7 +129,64 @@ class SmartSearchComponent extends React.Component {
...
+                )}
+              leftIcon={<ShoppingCartIcon />}
+            >
+              Add All Files to Cart
+            </Button>
+            <DownloadManifestButton
+              fileCount={this.props.viewer.repository.files.hits.total}
+              filters={this.props.filters}
+            />
+            {console.log(this.props)}
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/containers/SmartSearchPage.js]: can you remove this please

**Comment by cy** [comment]
the 'Browse Annotations' button has different styles on the Repository page vs. the SmartSearch. An idea, pull the 'Browse Annotations' into that row and then pull that row out into its own component. Gets rid of inconsistencies between the two pages and lessens code duplication.

File: src/packages/@ncigdc/components/ActionsRow.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import AnnotationsLink from '@ncigdc/components/Links/AnnotationsLink';
+import { CreateRepositoryCaseSetButton } from '@ncigdc/modern_components/withSetAction';
+import { fetchFilesAndAdd } from '@ncigdc/dux/cart';
+import { ShoppingCartIcon } from '@ncigdc/theme/icons';
+import DownloadManifestButton from '@ncigdc/components/DownloadManifestButton';
+
+export default compose(
+  connect(),
+  withRouter,
+)(({ filters, totalCases, dispatch, totalFiles, push }) => {
```
Comment (cy) [inline src/packages/@ncigdc/components/ActionsRow.js]: can you type this so it's clear `totalCases` and `totalFiles` are numbers and not arrays of objects.

```
import type { TGroupFilter } from '@ncigdc/utils/filters/types';

[...]
)(({ filters, totalCases, dispatch, totalFiles, push }: {filters: TGroupFilter, totalCases: number, totalFiles: number, dispatch: Function, push: Function}) => {
[...]
```
you get the idea

File: src/packages/@ncigdc/components/ActionsRow.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                    },
+                  ],
+                }),
+              },
+            });
+          }}
+        >
+          {'View '}
+          {totalCases.toLocaleString()}{' '}
+          {totalCases === 1 ? ' Case ' : ' Cases '}
```
Comment (cy) [inline src/packages/@ncigdc/components/ActionsRow.js]: can replace this with pluralize 
```
import pluralize from '@ncigdc/utils/pluralize';
[...]
pluralize('Case', totalCases)
```

**Comment by aishmitk** [comment]
👍

**Pull request merged**